gradle/verification-metadata.xml

```diff
@@ -280,6 +280,20 @@
       </trusted-keys>
    </configuration>
    <components>
+      <component group="android-studio" name="android-studio" version="2021.1.1.19">
+         <artifact name="android-studio-2021.1.1.19.windows.zip">
+            <sha256 value="27015ad5e1abb2586de52bc77a0b2b40e902db5dea718a4e2565b2d498e44cbb" origin="Artifact is not signed"/>
```

Comment:
Did you download those artifacts and manually verify the signature? Or did you run a task which downloaded all of the artifacts?

Reply:
I used values from: https://developer.android.com/studio/archive